emulsify-ds/gatsby-theme-emulsify-workspace
# Added MDX Starter

Addresses https://github.com/emulsify-ds/gatsby-theme-emulsify-workspace/issues/15

To test:
- Test that you think the new addition makes sense in the context
- Verify code looks good
- Run `yarn` inside root
- cd into new directory and run `yarn develop` and ensure everything runs ok
